packages/asc-ui/src/components/Focus/Focus.tsx

```diff
@@ -28,15 +31,15 @@ const Focus: React.FC<Props> = ({ children, onKeyDown, ...otherProps }) => {
   }, [myRef])
 
   return (
-    <div
+    <FocusStyle
```

Comment:
Could you elaborate a bit on how this solves the issue?

Reply:
If you place a prop like `zIndexOffset` directly on a `div` element, React will trigger a warning that it doesn't recognise that attribute. However, if you pass that prop to a `styled` component, `styled-components` handles it for you and will not place that prop directly on the element.

We only needs that prop in `ModalFocus` (which is an instance of `Focus` to make `showAboveBackDrop()` work) and we don't need it as a HTML attribute.

The `BackDropStyle` that uses the `zIndexOffset` is also a `styled.div` and that prop is not added to the `div` as well. 

The Modal component is still working. To proof this you can remove the `zIndexOffset` prop from `<ModalFocus />` and the Modal will fail to render.

Reply:
Ok that makes sense, although it's not very clear in the code so I can see someone removing it in the future by accident.